kodilla-exception/src/main/java/com/kodilla/exception/test/FlightFinder.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class FlightFinder {
+
+    public Flight findFlight(Flight flight) throws RouteNotFoundException {
+        Map<String,Boolean> arrivalAirports = new HashMap<>();
+        arrivalAirports.put("Warsaw", true);
+        arrivalAirports.put("Berlin", true);
+        arrivalAirports.put("London", false);
+
+        if (!arrivalAirports.containsKey(flight.getArrivalAirport())) {
+            throw new RouteNotFoundException(flight + " do not exists!");
+        } else if (arrivalAirports.get(flight.getArrivalAirport())) {
+            return Optional.of(flight).get();
```

Comment:
Ten kod sie nie skompiluje :( Ja napisalem inaczej.
Metida powinna zwracac Optional<Flight> i nie robisz tam get() tylko Optional.of(…)